code/game/objects/items/stacks/rods.dm

```diff
@@ -66,7 +61,7 @@
 			return
 
 		if(WT.remove_fuel(0,user))
-			var/obj/item/stack/material/steel/new_item = new(usr.loc)
+			var/obj/item/stack/material/new_item = new(usr.loc, 1, material?.type || /decl/material/solid/metal/steel)
```

Comment:
doesn't this need to use fancy shmancy new create object proc?

Reply:
Yeah I must have missed it.